kie-server-parent/kie-server-wars/kie-server/src/main/webc-resources/README.md

```diff
@@ -38,38 +42,29 @@ This instruction describes all steps to install KIE Server on Tomcat 9 standalon
     As a next step, configure a pooling data source, that relies on XA data source for creating new connections. 
     In the example, this data source is named “poolingXaDs”.
     
-    Edit TOMCAT_HOME/conf/context.xml and add following within Context tags of the file:
-    ``` 
-    <Resource 
+    Edit TOMCAT_HOME/conf/server.xml and add following within GlobalNamingResources tags of the file (e.g. after the UserDatabase Resource definition):
```

Comment:
@mswiderski please note that maven cargo creates all the declared datasource resources [1] in context.xml. Thus if we change it here, we don't test the exactly same configuration any more.

[1] https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-tests/pom.xml#L629

Reply:
@rsynek then the question is why that didn't work for @SimonDallaway?

Reply:
The only difference I see is the usage of custom XARecovery configuration. I quickly glanced over the webc distribution and test setup and I don't see it being used anywhere.